payment_source.go

```diff
@@ -10,9 +11,10 @@ import (
 // For more details see https://stripe.com/docs/api#create_charge
 type SourceParams struct {
```

Comment:
so I have concerns about how this approach: If we continue to add sources that the customers or charges API accept as hashes (eg. Card here), then this could get pretty bloated. My reasoning for allowing a Card struct here was to not lose the clarity and explicitness of knowing exactly whats needed to create a Card source